src/material/core/typography/_typography.scss

```diff
@@ -69,6 +69,37 @@
   @return map-merge($config, (font-family: $font-family));
 }
 
+// Whether a config is for the Material Design 2018 typography system.
+@function mat-private-typography-is-2018-config($config) {
+  @if map-get($config, headline-1) {
+    @return true;
+  }
+  @return false;
```

Comment:
Could we use `@return map-get($config, headline-1) != null` or is that just not how Sass works?

Reply:
It looks like you can do that